docusaurus.config.js

```diff
@@ -27,12 +27,23 @@ module.exports = {
       },
       items: [
         {
-          type: "doc",
-          docId: "intro",
+          href: "https://docs.harvesterhci.io/",
           position: "right",
           label: "Docs",
           className: "navbar__docs",
         },
+        {
+          to: "blog",
+          position: "right",
+          label: "Blog",
+          className: "navbar__blog",
+        },
```

Comment:
Instead of having our own blog, I may prefer to link to the https://www.suse.com/c/?s=harvester since most of the articles will be published on `suse.com` first.

Reply:
Changed the link for Blog as suggested